roles/mediamtx/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Ensure intel-media-va-driver-non-free is installed
+  ansible.builtin.package:
+    name: intel-media-va-driver-non-free
+    state: present
+  when: mediamtx_on_intel
+
+- name: Ensure ffmpeg is installed
+  ansible.builtin.package:
+    name: effmpeg
```

Comment:
Shouldn't this be `ffmpeg` without an `e`?

Reply:
ffmpeg is the package name.

Reply:
fixed